lib/entrypoint.sh

```diff
@@ -6,6 +6,26 @@ echo "::group::Internal logs"
 cd $INPUT_CWD
 echo "Running in $PWD."
 
+echo "---------------------------------------------"
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+echo $DIR
+ls -A $DIR
+which python3
+python3 --version
```

Comment:
Debug code?

Reply:
Yeah, basically.  Thought I'd leave it in for logging purposes, but I can take it out too if you want.

Reply:
Yes please